frontend/device/devicelistener.lua

```diff
@@ -51,24 +49,38 @@ end
 function DeviceListener:onShowWarmth(value)
     local powerd = Device:getPowerDevice()
     if powerd.fl_warmth ~= nil then
-        UIManager:show(Notification:new{
-            text = T(_("Warmth set to %1."), powerd.fl_warmth),
-            timeout = 1.0,
-        })
+        if powerd.fl_warmth_max == nil then
+            UIManager:show(Notification:new{
+                text = T(_("Warmth set to %1."), math.floor(powerd.fl_warmth)),
+                timeout = 1.0,
+            })
+        else
```

Comment:
This code path would affect some devices with a different scale. All of them are kobos. Not sure which of them but KOA1 for sure.

Reply:
You are right. Then I it would be good, if someone with an affected Kobo could test it.